src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -131,16 +131,24 @@
       + "`kafka`\n"
       + "    Must be a trio representing the Kafka coordinates, defaults to ``" + StringUtils.join(DEFAULT_KAFKA_PK_NAMES, ",") + "`` if empty.\n"
       + "`record_key`\n"
-      + "    If empty, all fields from the key struct will be used, otherwise used to whitelist the desired fields - for primitive key only a single field name must be configured.\n"
+      + "    If empty, all fields from the key struct will be used, otherwise used to extract the desired fields - for primitive key only a single field name must be configured.\n"
       + "`record_value`\n"
-      + "    If empty, all fields from the value struct will be used, otherwise used to whitelist the desired fields.\n";
+      + "    If empty, all fields from the value struct will be used, otherwise used to extract the desired fields.";
   private static final String PK_FIELDS_DISPLAY = "Primary Key Fields";
 
+  public static final String FIELDS_WHITELIST = "fields.whitelist";
```

Comment:
`pk.fields` is I think a more descriptive name because that config is a bit overloaded. It is a whitelist in some cases and in others it specifies the names to use (e.g. for `pk.mode=kafka` it must be exactly 3 names for topic/partition/offset, and for `pk.mode=record_key` where you expect the record key to be a primitive type you can assign the name to use for the primitive).